### PR TITLE
Add getDescriptor(s) tests

### PR DESCRIPTION
**Pull request by fokinv**
Added some getDescriptor(s) test, based on the getCharacteristic(s).

---

- [ ] `./mach build -d` does not report any errors
- [ ] `./mach test-tidy` does not report any errors
- [ ] These changes fix #__ (github issue number if applicable).

- [ ] There are tests for these changes OR
- [ ] These changes do not require tests because _____

### CODE REVIEW

**Comment by mmatyas** [comment]
lgtm

**Comment by dati91** [comment]
Can you split this into 2 commits?
The lock file should be in an "Update cargo" or something like that commit instead.

**Comment by dati91** [comment]
Otherwise looks good to me.

File: tests/wpt/mozilla/meta/MANIFEST.json [hunk 1]
```diff
@@ -6666,6 +6666,126 @@
...
+            "url": "/_mozilla/mozilla/bluetooth/getDescriptor/get-same-descriptor.html"
+          }
+        ],
+        "mozilla/bluetooth/getDescriptor/invalid-descriptor-name.html": [
+          {
+            "path": "mozilla/bluetooth/getDescriptor/invalid-descriptor-name.html",
+            "url": "/_mozilla/mozilla/bluetooth/getDescriptor/invalid-descriptor-name.html"
+          }
+        ],
+        "mozilla/bluetooth/getDescriptor/service-is-removed.html": [
```
Comment (zakorgy) [inline tests/wpt/mozilla/meta/MANIFEST.json]: characteristic-is-removed

File: tests/wpt/mozilla/tests/mozilla/bluetooth/getDescriptors/blacklisted-descriptors.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+'use strict';
+promise_test(t => {
+    window.testRunner.setBluetoothMockDataSet(adapter_type.blacklist);
+    return window.navigator.bluetooth.requestDevice({
+        filters: [{services: [blacklist_test_service_uuid]}]
+    })
+    .then(device => device.gatt.connect())
+    .then(gattServer => gattServer.getPrimaryService(blacklist_test_service_uuid))
+    .then(service => service.getCharacteristic(blacklist_exclude_reads_characteristic_uuid))
+    .then(characteristic => promise_rejects(t, 'SecurityError', characteristic.getDescriptors()))
```
Comment (zakorgy) [inline tests/wpt/mozilla/tests/mozilla/bluetooth/getDescriptors/blacklisted-descriptors.html]: NotfoundError

**Comment by dati91** [comment]
Seems OK.

**Pull request merged**